Userland/Applications/CrashReporter/main.cpp

```diff
@@ -301,6 +324,7 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
                 backtrace_text_editor->set_text(backtrace.text);
                 backtrace_text_editor->set_mode(GUI::TextEditor::Mode::ReadOnly);
                 backtrace_text_editor->set_should_hide_unnecessary_scrollbars(true);
+                full_backtrace.append(backtrace.text);
```

Comment:
How does this look with multiple threads? Should we delimit the different threads by using the backtrace title somehow?

Reply:
Could use a space between the `====` and the title (just to avoid it looking a bit cramped), after that it will presumably look quite OK.

Reply:
done!